src/net_processing.cpp

```diff
@@ -1406,6 +1410,62 @@ void PeerManagerImpl::FindNextBlocksToDownload(const Peer& peer, unsigned int co
     }
 }
 
+void PeerManagerImpl::TryDownloadingHistoricalBlocks(const Peer& peer, unsigned int count, std::vector<const CBlockIndex*>& vBlocks, const CBlockIndex *from_tip, const CBlockIndex* target_block)
```

Comment:
This look suspiciously similar to `FindNextBlocksToDownload`, is there no way to unify the two?

(sorry if I missed previous discussion on this)

Reply:
Thanks guys - I've taken https://github.com/bitcoin/bitcoin/commit/43b322188de227c1161fe3ee2dfd7b3a019cfc6a and made a cosmetic refactoring change.